src/lory.js

```diff
@@ -280,16 +281,18 @@ export function lory (slider, opts) {
         }
 
         if (rewindOnResize) {
-            index = 0;
+            if (!infinite) {
+                index = 0;
+            } else {
+                index = 1;
+            }
         } else {
             ease = null;
             rewindSpeed = 0;
         }
 
         if (infinite) {
-            translate(slides[index + infinite].offsetLeft * -1, 0, null);
-
-            index = index + infinite;
+            translate(slides[index].offsetLeft * -1, 0, null);
```

Comment:
Breaks sliders with multiple slides to scroll because you translate to the wrong slide if you hardly set this to 1